src/invoices/InvoicesList/InvoicesListFilters/InvoicesListFilters.test.js

```diff
@@ -19,9 +20,18 @@ const defaultProps = {
   activeFilters: {},
   applyFilters: jest.fn(),
   disabled: false,
+  fiscalYearOptions: [{
+    label: 'FY 2023',
+    value: 'FY 2023',
+  }],
 };
+
+const queryClient = new QueryClient();
+
 const renderInvoicesListFilters = (props = defaultProps) => render(
-  <InvoicesListFilters {...props} />,
+  <QueryClientProvider client={queryClient}>
```

Comment:
why not to mock `FiscalYearFilter` instead of wrapping in  QueryClientProvider? we do unit testing, `InvoicesListFilters` doesn't need to know how `FiscalYearFilter` works and what it requires

Reply:
It was throwing an error `"Error: No QueryClient set, use QueryClientProvider to set one"`. By adding the code above we have a green test cases

Reply:
reason why you wrapped is clear, there is better way to do this for unit test, plz look at the suggesting in the first comment

Reply:
Aha, I see what you mean.